### PR TITLE
feat/time_limit-Repository

### PR DESCRIPTION
**Pull request by 0xBinayak**
ISSUE  #23 

- [X] Added time_limit field to Repository model
- [X] Updated migrations
- [X] Updated Repository model class docstring

### CODE REVIEW

**Comment by 0xBinayak** [comment]
@djeck1432  PTAL

File: tracker/models.py [hunk 1]
```diff
@@ -131,6 +132,8 @@ class Repository(AbstractModel):
     name = models.CharField(max_length=DefaultModelValues.name_max_length)
     author = models.CharField(max_length=DefaultModelValues.author_max_length)
     link = models.URLField(max_length=DefaultModelValues.link_max_length)
+    time_limit = models.PositiveIntegerField(default=86400)
```
Comment (iamnovichek) [inline tracker/models.py]: Please, move this magic number to `traker/values.py` or `core/settings.py`

**Review by iamnovichek — CHANGES_REQUESTED** [review]

**Comment by 0xBinayak** [comment]
@iamnovichek  PTAL requested changes done

**Review by iamnovichek — APPROVED** [review]

**Comment by iamnovichek** [comment]
@binayak9932 Could you please sync your code with the main branch?

**Comment by 0xBinayak** [comment]
> @binayak9932 Could you please sync your code with the main branch .

Done

**Pull request merged**